### PR TITLE
Display Menu

### PR DESCRIPTION
**Pull request by SophieSjoberg**
As a customer
In order to select a dish
I should be able to see a list of dishes

https://app.zenhub.com/workspace/o/craftacademy/slow_food_sinatra_mvc_lions_and_crowns/issues/8

### CODE REVIEW

File: app/models/product.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Product < ActiveRecord::Base
+end
```
Comment (tochman) [inline app/models/product.rb]: I would like you to add validation for :name and :price

File: spec/_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+# describe  do
+#   pending "add some examples to (or delete) /Users/craftacademyone/projects/slow_food_sinatra_mvc_lions_and_crowns/Rakefile"
+# end
```
Comment (tochman) [inline spec/_spec.rb]: ?? Why is this here?

**Review by tochman — CHANGES_REQUESTED** [review]
Please see comments

File: app/models/product.rb (hunk 1 above)
Comment (SophieSjoberg) [reply app/models/product.rb]: Check!

File: spec/_spec.rb (hunk 2 above)
Comment (SophieSjoberg) [reply spec/_spec.rb]: Deleted now!

**Review by tochman — APPROVED** [review]
Great job. 👍🏼 Please make better use of the Git Pong protocol in future features.
